triton-vm/src/table/master_table.rs

```diff
@@ -1311,11 +1447,71 @@ mod tests {
     }
 
     #[test]
-    fn randomized_trace_tables_are_in_column_major_order() {
+    fn trace_tables_are_in_column_major_order() {
         let (_, _, main, aux, _) =
             master_tables_for_low_security_level(ProgramAndInput::new(triton_program!(halt)));
-        main.randomized_trace_table().column(0).as_slice().unwrap();
-        aux.randomized_trace_table().column(0).as_slice().unwrap();
+        main.trace_table().column(0).as_slice().unwrap();
+        aux.trace_table().column(0).as_slice().unwrap();
+    }
```

Comment:
Kinda silly of `ndarray` to allow the user to specify majority but not inspect it.

Reply:
I concur. This is the best I could come up with as a workaround. :shrug: